README.md

```diff
@@ -35,15 +37,13 @@ other resources in Triton.
 
 If you do not already have an account on Joyent Public Cloud, sign up [here](https://www.joyent.com/public-cloud).
```

Comment:
No more JPC.  This, along with any other Joyent things I see, might require a distinct bug.

Reply:
Also fixed. I just deleted that reference.